usr.bin/su/Makefile

```diff
@@ -0,0 +1,11 @@
+TOPDIR = $(realpath ../..)
+
+PROGRAM = su
+SOURCES = su.c suutil.c
+FORMAT-EXCLUDE = $(SOURCES)
+
+LDLIBS = -lutil
+
+#CPPFLAGS += -DALLOW_GROUP_CHANGE -DALLOW_EMPTY_USER
```

Comment:
Why this line is commented? Do you plan to use these flags later?

Reply:
Yes. Flag `DALLOW_GROUP_CHANGE` could be used in the future. `DALLOW_EMPTY_USER` is now used.